src/Windows.Core/ApiSets.cs

```diff
@@ -39,6 +39,11 @@ public static class ApiSets
         /// </summary>
         public const string api_ms_win_core_io_l1_1_1 = "api-ms-win-core-io-l1-1-1.dll";
 
+        /// <summary>
+        /// The "api-ms-win-core-file-l1-1-0.dll" constant.
+        /// </summary>
+        public const string api_ms_win_core_file_l1_1_0 = "api-ms-win-core-file-l1-1-0.dll";
+
         /// <summary>
         /// The "api-ms-win-core-file-l1-2-0.dll" constant.
         /// </summary>
```

Comment:
It's not used now, but since it's also a part of public api, so keeps it here.
https://github.com/dotnet/pinvoke/blob/master/src/Windows.Core/PublicAPI.Shipped.txt#L5243